lambdas/test-lambda/src/main/java/uk/gov/di/data/lep/classes/GroDeathEventDetails.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.di.data.lep.classes;
+
+import uk.gov.di.data.lep.enums.GroSex;
+
+import java.time.LocalDate;
+
+// This is the output
+public class GroDeathEventDetails {
```

Comment:
would it make sense to make this into a record class? and the `GroDeathEventEnrichedData`. Then we could do something like this for line 29 in the minimisation 
```
        var eventDetails = new GroDeathEventDetails(
            Config.enrichmentFields.contains(EnrichmentField.SOURCE_ID) ? enrichedData.sourceId : null,
            Config.enrichmentFields.contains(EnrichmentField.SEX) ? enrichedData.sex : null,
            Config.enrichmentFields.contains(EnrichmentField.DATE_OF_BIRTH) ? enrichedData.dateOfBirth : null,
            Config.enrichmentFields.contains(EnrichmentField.DATE_OF_DEATH) ? enrichedData.dateOfDeath : null,
            Config.enrichmentFields.contains(EnrichmentField.REGISTRATION_ID) ? enrichedData.registrationId : null,
            Config.enrichmentFields.contains(EnrichmentField.EVENT_TIME) ? enrichedData.eventTime : null,
            Config.enrichmentFields.contains(EnrichmentField.VERIFICATION_LEVEL) ? enrichedData.verificationLevel : null,
            Config.enrichmentFields.contains(EnrichmentField.PARTIAL_MONTH_OF_DEATH) ? enrichedData.partialMonthOfDeath : null,
            Config.enrichmentFields.contains(EnrichmentField.PARTIAL_YEAR_OF_DEATH) ? enrichedData.partialYearOfDeath : null,
            Config.enrichmentFields.contains(EnrichmentField.FORENAMES) ? enrichedData.forenames : null,
            Config.enrichmentFields.contains(EnrichmentField.SURNAME) ? enrichedData.surname : null,
            Config.enrichmentFields.contains(EnrichmentField.MAIDEN_SURNAME) ? enrichedData.maidenSurname : null,
            Config.enrichmentFields.contains(EnrichmentField.ADDRESS_LINE_1) ? enrichedData.addressLine1 : null,
            Config.enrichmentFields.contains(EnrichmentField.ADDRESS_LINE_2) ? enrichedData.addressLine2 : null,
            Config.enrichmentFields.contains(EnrichmentField.ADDRESS_LINE_3) ? enrichedData.addressLine3 : null,
            Config.enrichmentFields.contains(EnrichmentField.ADDRESS_LINE_4) ? enrichedData.addressLine4 : null,
            Config.enrichmentFields.contains(EnrichmentField.POSTCODE) ? enrichedData.postcode : null
        );
```

although maybe we should use a method instead of the long ternaries